Sources/Amplitude/Amplitude.m

```diff
@@ -1519,7 +1613,7 @@ - (NSDate *)currentTime {
 }
 
 - (void)printEventsCount {
```

Comment:
Do we need this function?

Reply:
In the header file, It says this method is used for debugging method to find out how many events are being stored locally on the device. So I think that's still valid?